homotopy-web/src/app/signature/item.rs

```diff
@@ -814,6 +814,11 @@ impl ItemView {
                 <button onclick={ctx.props().dispatch.reform(move |_| Action::Suspend(generator, generator))}>{"Suspend"}</button>
             </div>
         };
+        let strictification_button = html! {
```

Comment:
I think we should hide the button if it is invalid.

Reply:
Actually on second thoughts, checking if it is valid can be expensive (as it involves checking if the source and target are atomic), so maybe it's not the best idea.

Reply:
Agreed it's hard to check for it within the signature drawers.